Fig3_PyGMT_backgrounds.py

```diff
@@ -0,0 +1,44 @@
+import pygmt
+
+region = [-25, -13, 63.2, 66.7]
+
+fig = pygmt.Figure()
+with fig.subplot(
+    nrows=2,
+    ncols=2,
+    subsize=("12c", "8.5c"),
+    autolabel="(a)+o0.15c/0.3c+gwhite@30+v",  # vertical order of labels
+    margins=("0.3c", "0.2c"),
+    sharex="b",
+    sharey="l",
+    frame="WSrt",

```

Comment:
```suggestion
    sharex="b",
    sharey="l",
    frame="WSen",
```
I prefer to use "WSen" rather than "WSrt" here. The reasons are:

- `WSen` and `WSrt` makes minor differences for fancy frames (actually I almost see no differences)
- It's more straightforward to explain that upper-case letters (W and S) mean annotated axis, while lower-case letters (e and n) mean unannotated axis. Otherwise, we have to add more sentences explaining the difference between "rt" and "en", which is not the focus of this example.